mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+
```

Comment:
Once this one gets merged we may want to consider iteratively making this more strict but fine for now to just ask it to ignore imports